code/script.js

```diff
@@ -1,51 +1,199 @@
+    setTimeout(() => tempSelection(), 1000)
+  } else if (color === 'White') {
+    showMessage (`Let´s do some white laundry ${username}. At what temp?`, 'bot')
+    setTimeout(() => tempSelection(), 1000)
+  } else {
+    showMessage (`WOW, that´s daring ${username}. Choose your temp and give it a try`, 'bot')
+    setTimeout(() => tempSelection(), 1000)
+  }
+  inputWrapper.innerHTML =''
+}
+
+const tempSelection = () => {
+  inputWrapper.innerHTML = `
+  <select id="selectTemp">
+    <option value="" selected disabled>🌡️</option>
+    <option value="30">30°</option>
+    <option value="40">40°</option>
+    <option value="60">60°</option>
+  </select>
+  `
+const select=
+document.getElementById('selectTemp')
+.addEventListener('change', () => {
+showMessage(selectTemp.value + '°', 'user')
+
+  if (selectTemp.value === '30') {
+    setTimeout(() =>
+    showMessage (`OK ${username}, some sensitive laundry in the making!`, 'bot'), 1000)
+  } else if (selectTemp.value === '40') {
+    setTimeout(() =>
+    showMessage (`That´s cool ${username}!`, 'bot'), 1000)
+  } else {
+    setTimeout(() =>
+    showMessage (`Dirty laundry, in for washing ${username}!`, 'bot'), 1000)
+  }
+
+    inputWrapper.innerHTML =''
+    setTimeout(() => tumbleQuestion(), 3000)
+})
+}
+
+const tumbleQuestion = () => {
+  showMessage (`${username}, will you be using the dryer today?`, 'bot')
+  inputWrapper.innerHTML = `
+  <button id='yesBtn'> Yes</button>
+  <button id='noBtn'> No</button>
+  `
+  document.getElementById('yesBtn')
+    .addEventListener('click', () => {
+    showMessage('Yes' , 'user')
+    setTimeout(() => timeSet('Yes'), 1000)})
+  
+    document.getElementById('noBtn')
+    .addEventListener('click', () => {
+    showMessage('No' , 'user')
+    setTimeout(() => timeSet('No'), 1000)})
+}
+
+const timeSet = (tumbleAnswer) => {
+  if (tumbleAnswer === 'Yes') {
+    showMessage ('Washing and drying will take approx. 2 h' ,'bot')
+    setTimeout(() =>  
+    showMessage ('Room 1' , 'bot'), 2000)
+    setTimeout(() =>  
+    showMessage ('Enter code: 112233' , 'bot'), 2000)
+  } else {
+    showMessage ('Only washing will take approx. 1 h' ,'bot')
+    setTimeout(() =>  
+    showMessage ('Room 2' , 'bot'), 2000)
+    setTimeout(() =>  
+    showMessage ('Enter code: 223344' , 'bot'), 2000)
+  }
+  inputWrapper.innerHTML = ''
+  setTimeout(() => lastQuestion(), 5000)
+}
+
+const lastQuestion = () => {
+  inputWrapper.innerHTML = `
+  <button id='moreBtn'> I want to do some more washing!</button>
+  <button id='doneBtn'> I´m done for today!</button>
+  `
+  document.getElementById('moreBtn')
+  .addEventListener('click', () => {
+  showMessage ('Please reload the page and fill a new request!', 'bot')
+  setTimeout(() =>
+  showMessage ('❗️Don´t forget to save your code❗️', 'bot'), 2000)
+  setTimeout(() => end(), 2000)
+})
+  document.getElementById('doneBtn')
+  .addEventListener('click', () => {
+  inputWrapper. innerHTML =''
+  soundDone.play ()
```

Comment:
Well done for using two different sounds depending on which button the user presses.